CHANGELOG.md

```diff
@@ -0,0 +1,33 @@
+# Changelog
+
+Deprecated means no longer supported.
+Please update to the newest GA release.
+
+## 0.7.0
+
+The plugin is updated to Drone 0.5+ style (environment variables).
+
+**BREAKING CHANGES**
+
+- Consumes Drone 0.5+ style secrets, causing breaking changes with secrets and the GCP token across the board.
+Please read the documentation for reference usage.
+
+- Reduced available "default" environment variables to the plugin.
+To reference other environment variables, pass them via `vars`.
+  - `DRONE_BUILD_NUMBER`
+  - `DRONE_COMMIT`
+  - `DRONE_BRANCH`
+  - `DRONE_TAG`
+
+**FEATURES & IMPROVEMENTS**
+
+- `kubectl` is updated to 1.6.6.
```

Comment:
Should we update to even newer version?

Reply:
Yes, good point.

Reply:
Please see https://github.com/NYTimes/drone-gke/pull/57.

Reply:
CHANGELOG updated.